docs/conf/atlantic/2024/writing-day.rst

```diff
@@ -127,3 +127,43 @@ Project list
 ------------
 
 Submit your project using our `Writing Day project form <https://forms.gle/uTkWHV3fesyNQEyk9>`_ or send us a PR with your project info!
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+Audinix - Stash and Publish a User Guide (Session 1)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Join project organizer, Hank Lee, during the first Writing Day session.
+
+`Audinux <https://audinux.github.io/>`__ is a repository for FOSS music applications and plugins. We want to empower creative people and help them use Audinux to it's full potential! To do that, we need documentarians to help us create a user guide.
+
+Our Writing Day goal is to work together and create a user guide for Audinux. Creating a user guide will positively impact our users and encourage new users.
+
+Our plan is to create new documentation pages within the 'User Guide' templates. This means you can contribute using the GitHub web editor or using whatever Git-workflow from your favorite local writing setup and text editor. If you are unfamiliar with GitHub, we recommend using the `hackMD <https://hackmd.io/>`__ integration with GitHub.
```

Comment:
Delete: If you are unfamiliar with GitHub, we recommend using the `hackMD <https://hackmd.io/>`__ integration with GitHub.